test.sh

```diff
@@ -232,23 +232,34 @@ function nightly_functionality_stress_tests() {
 # of execution, especially to shake out AIO / thread registration issues.
 # #############################################################################
 function nightly_unit_stress_tests() {
-    local use_shmem=$1
+    local use_shmem=""
+    if [ $# -eq 1 ]; then
+        use_shmem="$1"
```

Comment:
This block of diffs will be seen in multiple places in this file.

This change now allows one to call the test-execution function, like so:

 `$ ./test.sh nightly_unit_stress_tests`
 
 
 i.e., we can now invoke this shell script to do dev-test on individual functions without shared memory configuration. (Previously that invocation would return a shell error.)